Eatery/Controllers/Eateries/Campus/CampusEateriesViewController.swift

```diff
@@ -13,7 +13,7 @@ import SwiftyUserDefaults
 import UIKit
 
 class CampusEateriesViewController: EateriesViewController {
-
+    
```

Comment:
Nit: remove newline

Reply:
I'm a little confused on what your comments are for removing white space.  I could be misunderstanding you but here it seems that you are telling me to delete whitespace at line 16.  However, this would cause there to be no line space between the class header on line 15 and the first variable declaration on line 17 (which would be moved to line 16).  I would understand if this was the conventional format, but all other files seem to leave 1 line of white space between the class header and the first line of actual code so I am a little confused.

Reply:
I guess I should have said remove whitespace. Line 16 should be a "\n"

Reply:
I believe changing my settings solved this!

Reply:
Perfect, now commit and push the updated file so it'll be on master when we merge it